### DISCUSSION TITLE
export to PNG is said as SVG in the hungarian translation (`blocks2image`)

### DISCUSSION THREAD

**Discussion by ghost** [ScratchAddons/ScratchAddons#7042] [Translation] [answered]
Unsure if I should make an issue (I don't really wanna sign up to Transifex) but I think this should be a good place to say it
If you are hovering over a block to export it as PNG or SVG, the context menu will display both as SVG:
![kép](https://github.com/ScratchAddons/ScratchAddons/assets/150537842/6ba90fb7-9645-41f4-9b14-3cc3c421ceb4)
Both are said as SVG, the above is plural but the below one isn't and this still happens for having a block under:
![kép](https://github.com/ScratchAddons/ScratchAddons/assets/150537842/67e3476b-f525-4031-972d-0556791dcd17)

**Comment by OregSamSas** [comment, marked as answer]
Actually, this translation mistake here would be my bad, so I think the tables turned :)
I corrected it in transifex, thank you for he report.

**Reply by ghost** [reply]
> Actually, this translation mistake here would be my bad, so I think the tables turned :)
> I corrected it in transifex, thank you for he report.

Thank you!
